.github/workflows/devcontainer.yml

```
@@ -0,0 +1,17 @@
+name: 'devcontainer'
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
```

```suggestion
  devcontainer-test:
```
it otherwise interfere with the normal required build job